github/github.go

```diff
@@ -735,6 +791,22 @@ func (r *AbuseRateLimitError) Error() string {
 		r.Response.StatusCode, r.Message)
 }
 
+func (r *AbuseRateLimitError) Is(target error) bool {
+	v, ok := target.(*AbuseRateLimitError)
+	if !ok {
+		return false
```

Comment:
Please add a unit test that covers this line.

Reply:
Done.